platforms/software/build-init/src/main/java/org/gradle/buildinit/tasks/InitBuild.java

```diff
@@ -78,9 +79,28 @@ public abstract class InitBuild extends DefaultTask {
     private ProjectLayoutSetupRegistry projectLayoutRegistry;
 
     /**
-     * The desired type of project to generate, defaults to 'pom' if a 'pom.xml' is found in the project root and if no 'pom.xml' is found, it defaults to 'basic'.
+     * Disables interactive dialog, ensuring no user input is required to complete the command.
+     * <p>
+     * This property can be set via the command-line options '--no-dialog' and '--dialog'.
      *
+     * @since 8.6
+     */
+    @Incubating
+    @Input
+    @Optional
+    @Option(option = "no-dialog", description = "Disable interactive dialog, use default values when none explicitly specified")
+    public abstract Property<Boolean> getNoDialog();
```

Comment:
❌ This needs to be added to `subprojects/docs/src/docs/dsl/org.gradle.buildinit.tasks.InitBuild.xml`

Reply:
Fixed